packages/web/src/components/track/DynamicTrackArtwork.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  useGetTrackById,
+  SquareSizes,
+  statusIsNotFinalized,
+  ID
+} from '@audius/common'
+import cn from 'classnames'
+
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import { useTrackCoverArt2 } from 'hooks/useTrackCoverArt'
+
+import styles from './DynamicTrackArtwork.module.css'
+
+export enum DynamicTrackArtworkSize {
+  /** 40x40 */
+  SMALL = 'small',
```

Comment:
I like what you've done here. Hope that we can harmonize this. FYI @Kyle-Shanks @dylanjeffers

Reply:
love this, totally need to do this for all images